products/claim-appeal-status/engineering/my-va-dashboard-discovery.md

```diff
@@ -0,0 +1,43 @@
+# Discovery: Claims and Appeals Access Issue on My VA Dashboard
```

Comment:
With this being a pretty short document (unlike the date discrepancy document) I think when the review is done you could just copy and paste this markdown into a comment on the Github issue then you can share a link to that comment in the thread. But it was great to have a place to comment. You could just close this PR after. But want to also hear what you prefer of course.